src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/help-windows.html

```diff
@@ -0,0 +1,31 @@
+<div>
+    Enable this option if the launched machine is a Windows VM. The Windows VM must have the OpenSSH server feature
+    installed on it, Java installed on it, and you must have configured an administrative user with a username and password.
+    <p>
+        The configured username and password should be set in the "Windows Username" and "Windows Password"
+        fields.
+    </p>
+    <p>
+        If you do not have the aforementioned pre-installed, you can use the following Startup Script to automatically set up the minimum requirements. In this example
+        the Windows username is "Build" and the password is "P@ssword2". <br />
```

Comment:
s/<br />/<br/>
Here and throughout the file.

Reply:
done